dockerfiles/startall.sh

```diff
@@ -53,7 +58,15 @@ done
 [[ -n ${INPUT_DOCKER_REPO} ]] && DOCKER_REPO=${INPUT_DOCKER_REPO}
 [[ -z ${LABEL} ]] && LABEL=$(docker image ls | fgrep "${DOCKER_REPO}/platform-core" | head -n 1 | awk '{print $2}')
 
-helm ${CMD} mycluster ../helm/hpcc/ --set global.image.root="${DOCKER_REPO}" --set global.image.version=$LABEL --set global.privileged=true ${restArgs[@]}
+if [[ -n ${PERSIST} ]] ; then
+  mkdir -p ${PERSIST}/dlls
+  mkdir -p ${PERSIST}/dali
+  mkdir -p ${PERSIST}/data
+  helm install localfile ../helm/examples/local/hpcc-localfile --set common.hostpath=${PERSIST} | grep -A100 storage > localstorage.yaml && \
```

Comment:
if you chose -u (upgrade) this would probably now fail?
should this be ${CMD} too?

Reply:
fixed